avaframe/tests/test_initializeDirs.py

```diff
@@ -0,0 +1,27 @@
+"""
+    Pytest for initialise directories
+
+ """
+
+#  Load modules
+import pathlib
+from avaframe.in3Utils import initialiseDirs as iD
+
+
+def test_initialiseRunDirs(tmp_path):
+    """ test initialising run directories """
+
+    # set directory
+    avaName = 'avaTest'
+    avaDirtmp = pathlib.Path(tmp_path, avaName)
+    modName = 'com1DFA'
+
+    # define results
+    workDirTest = avaDirtmp / 'Work' / modName
+    outputDirTest = avaDirtmp / 'Outputs' / modName
+
+    # call function to be tested
+    workDir, outputDir = iD.initialiseRunDirs(avaDirtmp, modName)
+
+    assert workDir == workDirTest
```

Comment:
maybe also test that this directory actually exists :)

Reply:
Done :)